app/views/GuidanceView.scala.html

```diff
@@ -45,12 +45,12 @@
             Html(<a class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-action="content_item 1" data-track-label="#who-should-claim" data-track-options="{&quot;dimension29&quot;:&quot;Who should claim&quot;}" href="#5">After you've disclosed</a>.toString)
         ))
 
+        @p(Html("If you’ve previously told us about underpaid Customs Duty or import VAT and you need to send us more information about your disclosure you can use this service. You’ll need to have your disclosure reference number."))
+
         <div class="govuk-inset-text">
             If you are only disclosing underpaid import VAT and use postponed VAT accounting, you should make adjustments on your VAT return to account for the underpaid import VAT. Do not use this service.
```

Comment:
This on the prototype is -
"If you're only disclosing underpaid import VAT and use postponed VAT accounting, you should account for the underpaid import VAT on your VAT return. Do not use this service."